packages/vue-router/src/router.ts

```diff
@@ -104,7 +103,6 @@ export const createIonRouter = (opts: IonicVueRouterOptions, router: Router) =>
   });
 
   const handleNavigateBack = (defaultHref?: string, routerAnimation?: AnimationBuilder) => {
-    // todo grab default back button href from config
```

Comment:
I noticed that Angular and React both get `backButtonDefaultHref` from the config (https://github.com/ionic-team/ionic-framework/blob/0a8a958cba02994ea0fefa265b17edddffe62e98/angular/src/directives/navigation/ion-back-button.ts#L30). However, it looks like we update `defaultHref` on `ion-back-button` with `backButtonDefaultHref`: https://github.com/ionic-team/ionic-framework/blob/0a8a958cba02994ea0fefa265b17edddffe62e98/core/src/components/back-button/back-button.tsx#L76

Are you able to check if this is needed in Ionic Vue? You can set `backButtonDefaultHref` in the list starter app and make sure that the back button shows up on the list page when no `default-href` is set. Just want to make sure we aren't missing something important here. Maybe not needed since we render the actual Web Component?


Somewhat related: Looks like `backButtonDefaultHref` is not documented: https://ionicframework.com/docs/angular/config

Reply:
Vue does indeed operate as expected with `backButtonDefaultHref` set 👍 The back-button stays visible, even after refreshing the page.

I'll see about getting `backButtonDefaultHref` added to the docs in a separate PR.

Reply:
Thanks for checking!